sdk/src/main/java/io/dapr/actors/runtime/DaprClientBuilder.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.actors.runtime;
+
+import okhttp3.OkHttpClient;
+import io.dapr.actors.*;
+
+/**
+ * Builds an instance of DaprAsyncClient or DaprClient.
+ */
+public class DaprClientBuilder {
```

Comment:
The class can be package private as its used only by actorRuntime. 
Update the class name, there are 2 classes with name DaprClientBuilder and it was confusing.